<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LT-jiangxm" name="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest" tests="28" failures="0" timestamp="2 Nov 2017 13:27:27 GMT" time="4.040" errors="1">
  <testcase name="addUserVbAccount" time="0.193" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest"/>
  <testcase name="addUserVbAccount" time="0.453" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest"/>
  <testcase name="addUserVbAccount" time="0.035" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest"/>
  <testcase name="addUserVbAccount" time="0.045" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest"/>
  <testcase name="addUserVbAccount" time="0.105" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest"/>
  <testcase name="addUserVbAccount" time="0.047" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest"/>
  <testcase name="addUserVbAccount" time="0.021" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest"/>
  <testcase name="addUserVbAccount" time="0.022" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest"/>
  <testcase name="addUserVbAccount" time="0.024" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest"/>
  <testcase name="addUserVbAccount" time="0.269" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest"/>
  <testcase name="addUserVbAccount" time="0.151" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest"/>
  <testcase name="addUserVbAccount" time="0.584" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest"/>
  <testcase name="addUserVbAccount" time="0.025" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest"/>
  <testcase name="addUserVbAccount" time="1.154" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest"/>
  <testcase name="addUserVbAccount" time="0.043" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest"/>
  <testcase name="addUserVbAccount" time="0.041" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest"/>
  <testcase name="addUserVbAccount" time="0.019" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest"/>
  <testcase name="addUserVbAccount" time="0.111" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest"/>
  <testcase name="addUserVbAccount" time="0.023" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest"/>
  <testcase name="addUserVbAccount" time="0.262" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest"/>
  <testcase name="addUserVbAccount" time="0.109" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest"/>
  <testcase name="addUserVbAccount" time="0.020" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest"/>
  <testcase name="addUserVbAccount" time="0.043" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest"/>
  <testcase name="addUserVbAccount" time="0.149" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest"/>
  <testcase name="addUserVbAccount" time="0.026" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest">
    <error type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest.addUserVbAccount(AddUserVbAccountTest.java:78)
	at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:46)
	at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:37)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- addUserVbAccount -->
  <testcase name="addUserVbAccount" time="0.028" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest"/>
  <testcase name="addUserVbAccount" time="0.019" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest"/>
  <testcase name="addUserVbAccount" time="0.019" classname="com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest"/>
</testsuite> <!-- com.llwallet.interfaces.test.api.test.personal.AddUserVbAccountTest -->
